1080i/SkinSettings.xml

```diff
@@ -1699,14 +1705,53 @@
                             <selected>Skin.HasSetting(dontuselandscapemovies)</selected>
                             <onclick>Skin.ToggleSetting(dontuselandscapemovies)</onclick>
                         </control>
+                        
+                        <!-- Indicators -->
                         <control type="radiobutton" id="9585">
                             <visible>ControlGroup(9100).HasFocus(9106)</visible>
                             <include>DefSettingsButtonGradient</include>
                             <label>37956</label>
                             <align>left</align>
                             <selected>!Skin.HasSetting(hide.markers)</selected>
                             <onclick>Skin.ToggleSetting(hide.markers)</onclick>
-                        </control>                        
+                        </control>
+                        <control type="radiobutton" id="9586">
+                            <visible>ControlGroup(9100).HasFocus(9106)</visible>
+                            <include>DefSettingsButtonGradient</include>
+                            <label>$LOCALIZE[37742]$LOCALIZE[37570]</label>
```

Comment:
Can you test if the string 575 works just as well? Cause the string 37570 is related to tv-shows i think. The german translation of 37570 is related to tv-shows. I think that 

`msgid "In Progress"` has to be `msgid "In Progress Tv Shows"`

Reply:
I think, this was my fault

Reply:
in the English/strings.po the 37575 is `random episodes`, is it?

Reply:
I meant the 575 string, that comes with the kodi language file: resource.language.en_us\resources\strings.po

Reply:
Oh, I got it, done!

Reply:
Also I change the `new` for `842`